x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -235,6 +247,8 @@ const savedObjectAuditTypes: Record<SavedObjectAction, EventType> = {
   saved_object_open_point_in_time: EventType.CREATION,
   saved_object_close_point_in_time: EventType.DELETION,
   saved_object_remove_references: EventType.CHANGE,
+  saved_object_collect_multinamespace_references: EventType.ACCESS,
```

Comment:
Thinking about this some more, I realized why I used a separate audit type --

This operation doesn't actually return any of the object's attributes, it just returns the object's metadata (what spaces it exists in, and what its inbound references are for the context of this operation).

From an auditing perspective, this would equate to false positives when you want to know who has accessed a given object. That's why we originally kept `saved_object_add_to_spaces` and `saved_object_delete_from_spaces` separate, because it's not really equivalent to `saved_object_update`.

So I'm leaning towards leaving this as-is, WDYT?

Reply:
I'll buy that argument - thanks for giving this some more thought!

Reply:
can we document this decision in code so that future-me doesn't forget why we have this distinction?